moveit_core/online_signal_smoothing/src/acceleration_parameters.yaml

```diff
@@ -0,0 +1,16 @@
+online_signal_smoothing:
```

Comment:
In accordance with my CMake comments, I think `acceleration_filter_parameters.yaml` would be better filename here. Only "acceleration" is too generic IMO

Reply:
Renamed